test/examples/ucp_hello_world.c

```diff
@@ -509,7 +532,7 @@ int parse_cmd(int argc, char * const argv[], char **server_name)
 {
     int c = 0, index = 0;
     opterr = 0;
-    while ((c = getopt (argc, argv, "wfbn:p:h")) != -1) {
+    while ((c = getopt (argc, argv, "wfbn:p:s:h")) != -1) {
```

Comment:
@guy101 yes, you right, this is the issue for previous developers (including me). Could you please fix this minor issue?